src/pages/messages/components/NviCandidateListItem.tsx

```diff
@@ -39,7 +42,7 @@ export const NviCandidateListItem = ({ nviCandidate, currentOffset }: NviCandida
   const myApproval = nviCandidate.approvals.find((approval) => approval.institutionId === user?.topOrgCristinId);
 
   const candidateLinkState = {
-    candidateOffsetState: { currentOffset, nviQueryParams: nviParams },
+    candidateOffsetState: { currentOffset, nviQueryParams },
```

Comment:
This is the fix for the arrow-problem (where we could navigate between tickets using arrows, but we navigated in the wrong "list", so items would appear that wasn't out search result)